src/Workspaces/Core/Portable/Workspace/Solution/DocumentState.cs

```diff
@@ -330,15 +324,15 @@ private DocumentState SetParseOptions(ParseOptions options)
             var newTreeSource = CreateLazyFullyParsedTree(
                 this.textAndVersionSource,
                 this.Id.ProjectId,
-                GetSyntaxTreeFilePath(this.info),
+                GetSyntaxTreeFilePath(this.Attributes),
```

Comment:
ah. now we hold onto attributes, not info. nice.

Reply:
Yes, turns out you introduced the Attributes type as a much better fix for the TextLoader problem awhile back. So this code was still creating new infos to only use the Attributes in the end... :smile: